edi_oca/templates/exchange_chatter_msg.xml

```diff
@@ -13,7 +13,7 @@
                     t-att-data-oe-model="exchange_record._name"
                     t-att-data-oe-id="exchange_record.id"
                 >
-                    <span t-field="exchange_record.name" />
+                    <span t-field="exchange_record.identifier" />
```

Comment:
why identifier ? I was expecting `related_name`

Reply:
this msg is posted on the related record to say that it has been created or update BY the exc record